app/page.tsx

```diff
@@ -0,0 +1,65 @@
+import '../css/style.css'
+import Link from 'next/link'
+
+export default function Page () {
+  return (
+    <html>
+      <title>206 Seattle Bike Polo</title>
+      <body>
+        <div id='main'>
+          <div className='tournament-nav-container'>
+            <Link href='/'>
```

Comment:
This will need to be reworked to be used by both `/` and `/tournament` routes (aka a Next "Layout"). More info here: https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#layouts